package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "webpack-exercise",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
```

Comment:
- [ ] Please, add `"dev": "webpack --mode development",` to open the development mode. This will help you to test the dev mode in your application. After that try it to see if there's any problem with this. 🐱